src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -93,8 +99,18 @@ class DSLCommandLineArguments : Callable<Int> {
         }
     }
 
+    @Option(names = ["-f", "--show_file_in_finder"], description = ["Show the output file in finder."])
```

Comment:
As in documentation finder is a MacOS application, something like "open file manager to generated file" would be better

Reply:
Update function name accordingly too